modules/administration-guide/partials/con_gateway.adoc

```diff
@@ -1,7 +1,16 @@
 [id="gateway_{context}"]
 = Gateway
 
-The {prod-short} gateway is a Traefik instance applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource.
+The {prod-short} gateway has three main roles:
+
+* Routing requests. It uses link:https://github.com/traefik/traefik[Traefik].
+
+* Ensuring authentication with OpenID Connect (OIDC). It uses {link-oauth2-proxy}.
+
+* Applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource. It uses {link-kube-rbac-proxy}.
+
+{link-oidc} is the default OIDC issuer. The {prod-short} operator deploys it with preconfigured static users. It can serve as a bridge to third party {identity-provider}, such as GitHub.
```

Comment:
```suggestion
{link-oidc-issuer} is the default OIDC issuer. The {prod-short} operator deploys it with preconfigured static users. It can serve as a bridge to third party {identity-provider}, such as GitHub.
```